by-language/php-pdo/README.rst

```diff
@@ -0,0 +1,47 @@
+.. highlight: console
+
+##############################################################
+Connect to CrateDB and CrateDB Cloud using PHP PDO (PDO_PGSQL)
+##############################################################
+
+
+*****
+About
+*****
+
+This is an example program demonstrating CrateDB's `PostgreSQL wire protocol`_
+compatibility by exercising a few basic SQL statements using PHP's PDO database
+adapter interface, see `PHP Data Objects`_. Specifically, it uses the `PDO_PGSQL`_
+driver.
```

Comment:
Shouldn't we at least mention our `CratePDO` and what the difference is (in short)  or why one should use one over the other?

Reply:
Same here, see my previous comment about a warning that this is experimental and such may not completely work

Reply:
I've added a corresponding note to both README documents.

> Also, please note that support for this driver is experimental. The [CrateDB PDO driver](https://crate.io/docs/pdo/) is the official CrateDB PHP client supported by Crate.io.